rmarkdowns/Week_4.Rmd

```diff
@@ -261,7 +261,107 @@ For your challenges this week, create the necessary code chunks below. You can c
 
 **Challenge 3: Create an interactive map using the `leaflet` function of your target area and target genus/species.** 
 
+```{r}
+inat_obs_df <- get_inat_obs(taxon_id = 1455922, 
+                            place_id = 7055, 
+                            quality = "research", #specifies research grade only
+                            geo = TRUE, #Specifies that we want geocoordinates
+                            maxresults = 100) #Limits results
+
+save(inat_obs_df, file = "egypt_LaughingDove_inat.Rdata")
+
+inat_obs_sf <-  inat_obs_df %>% 
+  select(longitude, latitude, datetime, common_name, scientific_name, image_url, user_login) %>% 
+  st_as_sf(coords=c("longitude", "latitude"), crs=4326)
+
+dim(inat_obs_sf)
+
+ggplot() +
+  geom_point(data = inat_obs_df, aes(x = longitude, y = latitude, color = common_name, text = common_name)) 
+
+egypt_bb <- getbb("Egypt")
+egypt_bb
+
+
+
+egypt_streets <- egypt_bb %>%
+  opq() %>%
+  add_osm_feature("highway", c("motorway", "primary", "secondary", "tertiary")) %>%
+  osmdata_sf()
+
+
+egypt_small_streets <- egypt_bb %>%
```

Comment:
Since Egypt is such a large area, we don't need the detail of all of the minor streets. Also, my computer couldn't pull this layer because it was too much data.